framework/src/multiapps/TransientMultiApp.C

```diff
@@ -398,8 +394,8 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
               if (!caught_up)
                 throw MultiAppSolveFailure(name() + " Failed to catch up!\n");
             }
-          }
-        }
+          } // if (!ex->lastSolveConverged())
+        }   // if (auto_advance)
```

Comment:
Remove the comments, we do not do that. A decent editor can take you to the matching `{` and back.